src/main/java/main/MainClass.java

```diff
@@ -3,13 +3,12 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
  
-//this annotation tells that this class is the starting point of our application 
-//the argument passed are the packages which we would like Spring to scan for Spring components. 
-//Refer: https://smarterco.de/java-spring-boot-mvc-ontroller-not-called/
 @SpringBootApplication(scanBasePackages = {"entity", "controller", "dbaccessor", "exceptionhandler", "helper", "main"})
 public class MainClass {
     public static void main(String[] args) {
-        //to start running the application
+        /*
+         * Starts running the application.
+        */
```

Comment:
FYI: As @sankalpbansal-google mentioned, you can continue to use `// ...` style for such small implementation comments.

Reply:
Done